bluesky/callbacks/best_effort.py

```diff
@@ -529,6 +529,46 @@ def stop(self, doc):
         self.check_visibility()
         super().stop(doc)
 
+class BaselinePrinter(Callback):
+    def __init__(self, start_doc, file=sys.stdout):
+        # We accept a start_doc but discard it.
+        ...
+        self._baseline_enabled = True
```

Comment:
This switch can be removed entirely (for the reason as my similar comment about `HeadingPrinter`).